deeprank_gnn/domain/forcefield/__init__.py

```diff
@@ -3,6 +3,8 @@
 
 import numpy
 
+from deeprank_gnn.models.structure import Atom
```

Comment:
Are there not any libraries out there that implement these kind of forcefields? I would assume people need to compute this quite often in a range of different domains?

Reply:
We should talk about that next meeting.

Reply:
We'll talk about this at today's meeting